src/main/java/api/IBuilder.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package api;
+
+public interface IBuilder {
```

Comment:
if you are going to have more implementation of IBuilder, you could keep this interface. in case only one class will implement this interface. we can remove this class